src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -185,6 +185,10 @@
       <Context>BrowseObject</Context>
     </PropertyPageSchema>
 
+    <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)PotentialEditorConfigFiles.xaml">
+      <Context>File</Context>
```

Comment:
"ProjectSubscriptionService", "File" will work - but that context is supposed to be for the project tree.

Reply:
Fixed.